### ISSUE TITLE
unproper layout on mobile view

### ISSUE THREAD

**Issue by ol3mon** [creecros/Customizer#127]
Hello,

I got an unproper layout on mobile view while using Customizer plugin.
The board view on mobile is taking like 30% to 50% of view port width.
Chrome inspector is reporting that html/body are 414px width. I can't figure out what's causing the vertical white space.
I tried with an uploaded CSS theme file to force html/body width:100vw!important, but it didnt fixed the issue.

App/Plugin version used to reproduce the issue:

- kanboard v1.2.18/Customizer v1.13.6 (production)
- kanboard v1.2.20/Customizer v1.13.7 (test)

Devices/Browsers used to reproduce the issue:

- Samsung Galaxy S8/Chrome browser
- Samsung Galaxy S8/Samsung Internet browser
- Emulated mobile (dev tools)/Chrome browser

Here are screenshots to illustrate the issue.
I've colored background on the fly using chrome dev tools for more visibility.

- html {background-color: red;}
- body {background-color: aqua;}

`Screenshot without Customizer:`

![kanboard_no_customizer_mobile_view_colored_backgrounds_00](https://user-images.githubusercontent.com/74542472/136865432-6b4cf5e2-1f5e-4d7a-a29b-2fbf339456a7.PNG)

`Screenshot with Customizer:`

![kanboard_customizer_mobile_view_bug_colored_backgrounds_01](https://user-images.githubusercontent.com/74542472/136865448-035546f0-07de-477b-a9f8-503bb5f864cc.PNG)

**Comment by ol3mon** [comment]
Hi, I was off of kanboard for a while.
Today I played a bit with the customizer overriding layout.php

I fixed the issue by editing (at the head tag) the following line:

`<meta name="viewport" content="width=device-width">`

by:

`<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">`

I hope this can help others. And thanks @creecros for the hint!
